OpenVote/war/goodbye.jsp

```diff
@@ -9,19 +9,20 @@
   </head>
   
   <body>
+  	<div class="page-header">
+ 		<img style="display: inline; width:50px;height:50px" src="http://www.clker.com/cliparts/B/B/Z/v/Z/Q/star-th.png" class="img-responsive" alt="Red Star">
+  		<h2 style="display: inline"> OpenVote </h2>
+  	</div>
   
     <div class="container">
-  		<h2>OpenVote</h2>
-		<div class="container">
-		  	<div class="jumbotron">
-		    	<h1>Thank you for voting!</h1>      
-		    	<p>Be sure to visit OpenVote.org later and verify your vote using the vote ID given to you.</p>
+		<div class="jumbotron">
+			<h1>Thank you for voting!</h1>      
+		   	<p>Be sure to visit OpenVote.org later and verify your vote using the vote ID given to you.</p>
 		  	
-		    	<form action="/results" method="post">
-		    		<div><input type="submit" value="Click here to go to Live Results!"/></div>
-		    	</form>
-		    </div>      
-		</div>
+		    <form action="/results" method="post">
+		    	<div><input class="btn btn-default" type="submit" value="Click here to go to Live Results!"/></div>
+		    </form>
```

Comment:
Would be nice if we also had a button here to directly logout, in case I don't want to take the time to look at the live results